subsys/bluetooth/controller/ll_sw/ull_llcp_common.c

```diff
@@ -570,7 +572,20 @@ static void rp_comm_send_rsp(struct ll_conn *conn, struct proc_ctx *ctx, uint8_t
 		}
 		break;
 	case PROC_MIN_USED_CHANS:
+		/* Spec says (5.2, Vol.6, Part B, Section 5.1.11):
+		     The procedure has completed when the Link Layer acknowledgment of the
+		     LL_MIN_USED_CHANNELS_IND PDU is sent or received.
+		   In effect, for this procedure, this is equivalent to RX of PDU
+		*/
+		/* So we inititate a chmap update procedure, but only if acting as central, just in case ... */
+		if (!conn->lll.role) { // && ull_conn_lll_phy_active(conn, ctx->data.muc.phys)) {
```

Comment:
Check `role` againt `BT_HCI_ROLE_MASTER`

Reply:
yeap

Reply:
and bring back the reminder of the if clause - that was temp commented out...